src/main/java/pl/edu/mimuw/BigInt.java

```diff
@@ -9,34 +9,133 @@
   private final boolean isPositive;
 
   public BigInt(String number) {
-    throw new IllegalStateException("TODO task 1: parse number from valid String");
+    int iter;
```

Comment:
takie też mogą być `final` polecam tak pisć, to dajesz świadectwo, że jest to 1 przypisanie tylko